trendshop/trendshop_website/views.py

```diff
@@ -1,9 +1,23 @@
 from django.shortcuts import render, redirect, get_object_or_404
 from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required, user_passes_test
+from .forms import ProductsForm
+
 
 def home(request):
-    return render(request, 'website.html')
+    return render(request, 'website.html',)
 
-def become_seller(request):
+def is_seller(user):
+    return user.is_authenticated and user.userprofile.role == 'seller'
 
```

Comment:
this function check return true or false based on user authentication right ?
then move this function to permissions.py file